views/posters.mustache

```diff
@@ -0,0 +1,13 @@
+{{> partials/header}}
+<h1>Paste a Poster!</h1>
+<p>Help spread the word about Plateletdonors.org by printing our poster on your Printer. Please select the poster in the language you want to print it . You can immediately paste these posters in your Office Notice Board, Residential area Notice Board, Hospital, Blood Bank or any other Public Place. This act of yours will help save many lives. If you are an NGO / Corporate you can even download the Corel Draw file and get it printed and distribute it in your city.</p>
+<ul style="list-style-type:decimal">
```

Comment:
Inline CSS needs to a SCSS file. Add a class like so -  .list.list__unnumbered and add this rule there.